RussianFilter/sections/specific.txt

```diff
@@ -51,6 +51,7 @@ dota2.ru,meteo.ua,klops.ru,crazymailing.com,lotzon.pro,novosti-n.org,prokazan.ru
 !
 !
 !
+m.sport-express.ru##.mb_10
```

Comment:
Too wide rule.
It's better to use `m.sport-express.ru##div[style*="width:325px;"][style*="height: 250px;"]` and please add it to mobile filter > specific_web.txt

Reply:
Fixed.